### PR TITLE
Update README for Follow-Along Hardware

### PR DESCRIPTION
**Pull request by cdwilson**

### CODE REVIEW

**Review by ChrisGammell — COMMENTED** [review]
Please leave a link in the new section back to the project page ("For more details on this setup and photos of the result, check out the project page")

File: README.rst [hunk 1]
```diff
@@ -273,3 +368,10 @@ recommend the following workflow to pull in future changes:
 .. _MikroE Weather Click: https://www.mikroe.com/weather-click
 .. _MikroE HVAC Click: https://www.mikroe.com/hvac-click
 .. _Reference Design Template: https://github.com/golioth/reference-design-template
+.. _Air Quality Monitor Project Page: https://projects.golioth.io/reference-designs/air-quality-monitor
+.. _nRF9160 DK Follow-Along Guide: https://example.com
```
Comment (cdwilson) [inline README.rst:372]: @cdwilson update this with the final link after https://github.com/golioth/projects/pull/40 is merged

File: README.rst [hunk 2]
```diff
@@ -2,90 +2,101 @@
...
+   * - Hardware
+     - Zephyr Board
+     - Follow-Along Guide
 
-   cd ~/golioth-reference-design-air-quality
-   west init -m git@github.com:golioth/reference-design-air-quality.git .
-   west update
-   west zephyr-export
-   pip install -r deps/zephyr/scripts/requirements.txt
+   * - .. image:: https://placehold.co/600x400?text=nrf9160dk_nrf9160_ns
```
Comment (cdwilson) [inline README.rst:49]: @cdwilson fix this before merging

**Review by MarkoPura — APPROVED** [review]
I like it a lot! LGTM!
Let's wait for @szczys blessing before merging.

**Pull request merged**